packages/types/src/lib/interfaces.ts

```diff
@@ -763,6 +763,7 @@ export interface SessionKeyPair {
 export interface AuthMethod {
   authMethodType: number;
   accessToken: string;
+  pubkey?: string;
```

Comment:
Should we separate the idea of `Auth Method` and data used to register auth methods?
I think it would help as the `AuthMethod` doesnt technically have this concept. in my previous PR i added:
```
export interface RelayerRequest {
  authMethodType: number;
  authMethodId: string;
  // used only in Webauthn credential registration
  authMethodPubKey?: string;
}
```
I think we should introduce it again as it is used in the new relay route.

Reply:
sgtm, added `IRelayRequestData`